packages/multichain/tsconfig.build.json

```diff
@@ -3,8 +3,16 @@
   "compilerOptions": {
     "baseUrl": "./",
     "outDir": "./dist",
-    "rootDir": "./src"
+    "rootDir": "./src",
+    "resolveJsonModule": true
```

Comment:
What are we using this for? I didn't see a JSON file being imported, but I might have missed it.

Reply:
api-specs imports json under the hood

https://github.com/MetaMask/core/pull/4784/files#diff-029fca0a68209f13ce287f80b2930bcf831f237e83c4cba752317d679a64b682R1

Reply:
Ah, gotcha, makes sense.